cmd/manager/main.go

```diff
@@ -13,11 +12,14 @@ import (
 	"github.com/spf13/pflag"
 	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
 	halkyon "halkyon.io/api"
+	"halkyon.io/operator-framework"
 	"halkyon.io/operator/pkg/controller/capability"
 	"halkyon.io/operator/pkg/controller/component"
-	"halkyon.io/operator/pkg/controller/framework"
 	"halkyon.io/operator/pkg/controller/link"
+	capability2 "halkyon.io/plugins/capability"
```

Comment:
Must this var named as `Capability2` and not just as `Capability` ?

Reply:
This is not a var but an import alias. It could maybe be named something else, I need to check.